presets/4.5/tune/karate/karate_whoop_late_2024.txt

```diff
@@ -0,0 +1,183 @@
+#$ TITLE: Karate whoop late 2024
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: TUNE
+#$ STATUS:  EXPERIMENTAL
```

Comment:
extra space. This one is important when you search for all experimental presets to turn them community